### PR TITLE
feat: add new colors on themes

### PR DESCRIPTION
**Pull request by theo-mesnil**
Signed-off-by: Théo Mesnil <mesniltheo@gmail.com>

#1247 

This PR start rework on colors and dark theme.
Some components are not complety done, this going to be fixed in on other pull request.

### CODE REVIEW

File: docs/components/Colors.js [hunk 1]
```diff
@@ -1,41 +1,43 @@
...
-                />
-                <Box pt="sm" px="sm" textAlign="center">
-                  <Box color="dark.900" fontWeight="medium">
-                    {key}
-                  </Box>
-                  <Box color="nude.800" fontSize="meta1" mt={3}>
-                    {depth}
-                  </Box>
-                </Box>
+        <Card.Body as={Grid} templateColumns="repeat(auto-fit, minmax(250px, 1fr));" gap="xl" backgroundColor={isLight && 'dark.900'}>
```
Comment (cnairiwttj) [inline docs/components/Colors.js:15]: It's really a tiny detail, but is the ";" necessary here ? `templateColumns="repeat(auto-fit, minmax(250px, 1fr));"` ?

File: docs/components/Header/NavBar/styles.js [hunk 2]
```diff
@@ -18,23 +18,26 @@ export const NavBar = styled.ul`
 `
 
 const activeItem = css`
+  opacity: 1;
   color: dark.900;
 
   @media (min-width: md) {
-    color: light.900;
+    opacity: 1;
+    color: white;
```
Comment (cnairiwttj) [inline docs/components/Header/NavBar/styles.js:26]: So now for light colors we use white + opacity value ?

File: docs/components/Header/styles.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ export const Header = styled.header(
     justify-content: space-between;
     width: 100%;
     height: ${headerHeight};
-    background-color: dark.800;
+    background-color: black;
```
Comment (cnairiwttj) [inline docs/components/Header/styles.js:16]: Same question for dark colors => we use black + opacity ?

File: scripts/watch.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ const handleFileChange = () => {
   const [dir, packageFolder, file] = args[0].split('/')
   const packagePath = path.join(ROOT_PATH, dir, packageFolder)
   const date = `${new Date().getHours()}:${new Date().getMinutes()}`
-
+  console.log(dir, packagePath)
```
Comment (cnairiwttj) [inline scripts/watch.js:14]: Is it wanted to keep this console.log here?

**Review by cnairiwttj — APPROVED** [review]

**Review by mleralec — APPROVED** [review]

File: docs/components/Header/NavBar/styles.js (hunk 2 above)
Comment (theo-mesnil) [reply docs/components/Header/NavBar/styles.js:26]: yes, white stay white on dark mode, same for the header is in black and stay black in dark mode

**Pull request merged**